app/models/question.rb

```diff
@@ -31,6 +31,10 @@ class Question < ApplicationRecord
   ANSWERS_TRUE_FALSE = [['1', 1], ['2', 2]].freeze
```

Comment:
I don't understand what these constants are for.  I presume they are defaults, but their purpose should be explained by comment(s).